### PR TITLE
style: Prettier the HTML & CSS files

### PR DESCRIPTION
**Pull request by abdonrd**
#### Description of what you did:

Prettier the HTML files.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Please confirm that your PR fulfills these requirements

- [x] Multiple commits are squashed into one commit.
- [x] The commit message follows [Conventional Commits](https://github.com/IBM/kui/blob/master/CONTRIBUTING.md#conventional-commits), which allows us to autogenerate release notes; e.g. `fix(plugins/plugin-k8s): fixed annoying bugs`
- [x] All npm dependencies are pinned.

### CODE REVIEW

**Comment by abdonrd** [comment]
@starpit please, check if it is right: https://github.com/IBM/kui/pull/2043/commits/c3a0742c40dea265e574ce2ab7eaa07ef4e36a76

**Comment by abdonrd** [comment]
Also add CSS files now.

**Comment by myan9** [comment]
Maybe squash the commits the PR's ready? Thanks!

**Comment by abdonrd** [comment]
@myan9 done!

**Comment by abdonrd** [comment]
@myan9 and green Travis!

File: plugins/plugin-tutorials/web/html/index.html [hunk 1]
```diff
@@ -34,26 +36,30 @@ <h2 class="tutorial-content-extras-title">Learn More</h2>
...
                     <div class="bx--structured-list-thead">
                       <div class="bx--structured-list-row bx--structured-list-row--header-row">
                         <!-- header row -->
                       </div>
-                    </div> <!-- structured list thead -->
-                    <div class="bx--structured-list-tbody">
-                    </div> <!-- structured list tbody -->
-                  </div> <!-- structured list -->
+                    </div>
+                    <!-- structured list thead -->
```
Comment (starpit) [inline plugins/plugin-tutorials/web/html/index.html]: the point of these comments was to help understand which element was being closed. it seems like, after moving the comment to the next line, the comments need comments....

Comment (abdonrd) [reply plugins/plugin-tutorials/web/html/index.html]: Ouch! We can't customize it. Do you prefer to remove Prettier from HTML files?

Comment (starpit) [reply plugins/plugin-tutorials/web/html/index.html]: i think we either need to remove these "closing comments" or not move them to a new line. with this PR as it stands, the closing comments become quite confusing

Comment (abdonrd) [reply plugins/plugin-tutorials/web/html/index.html]: We can not avoid moving it, Prettier does not have that configuration.

Do you prefer to remove comments or remove Prettier from HTML files?

Comment (starpit) [reply plugins/plugin-tutorials/web/html/index.html]: let's remove those closing comments, thanks!

Comment (abdonrd) [reply plugins/plugin-tutorials/web/html/index.html]: Updated!

**Pull request merged**